### ISSUE TITLE
Components add ARIA attributes not allowed to divs

### ISSUE THREAD

**Issue by itsmemarioss** [lumada-design/hv-uikit-react#3537]
### Which version of NEXT UI Kit are you opening this bug for?

v5.x

### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

`HvBadge` and `HvDropdownMenu` add aria attribute do a `div` without role which produces `aria-allowed-attr` issues in the accessibility tests.

### HvBadge

```
<HvBadge
    count={4}
    icon={<Alert />}
    showCount={true}
    id="badge"
  />
```

![badge](https://github.com/lumada-design/hv-uikit-react/assets/8560411/170bb074-335c-4ef4-92ad-3a019dd16264)

`divs` should not be named: https://www.w3.org/TR/html-aria/#dfn-naming-prohibited

### HvDropDownMenu

```
  <HvDropDownMenu
    id="dropdown"
    placement="right"
    keepOpened={false}
    dataList={[
      {
        label: "label",
        id: "id"
      },
    ]}
  />
```

![dropdown](https://github.com/lumada-design/hv-uikit-react/assets/8560411/abed2973-3ab2-4330-a31c-c7cbe7131c2d)

aria-expanded is not allowed

### Expected behavior 🤔

Do not include not allowed aria attributes.

### Steps to reproduce 🕹

And/or the steps to reproduce:

1. add the components
2. inspect HTML

### Context 🔦

The app shell team is tracking down accessibility errors:

https://hv-eng.atlassian.net/browse/TAP-9547

some of them come from the components reported.

### Your Environment 🌎

Device: [Laptop]
OS: [MacOS Ventura 13.4.1]
Browser [chrome]
Version [15.0.5790.98 (Official Build) (x86_64)]

**Comment by zettca** [comment]
Hello @itsmemarioss 👋 thanks for reporting this!

To me this seems to be 3 distinct issues:

1. Our icons contain the `name` attribute, which they shouldn't
2. The role of the icons. The user of the icon is responsible for this
  a. In the `HvBadge` case, that would be you - you should be passing a role/labels to the `Alert` icon
  b. in the `HvDropdownMenu`, UI Kit is responsible for it, we should be adding a role there
3. `HvDropdownMenu` is missing the appropriate role

- `1.` and `2.b.` are addressed here https://github.com/lumada-design/hv-uikit-react/pull/3555
- you should be handing the role in `2.a`
- We'll open a separate issue to address `3.`

How does that sound?

**Comment by zettca** [comment]
> Ok got it. I will add the role. IMHO it is strange adding `aria-label` to the div without a role since it produces an accessibility error as I said. I would suggest not adding the `aria-label` to the badge and leaving it to the user to define this in order to avoid this strange situation. People tend to believe and accept the default behavior as the correct one. And in this case, it is not. Besides, just having the number in as an `aria-label` may not be that helpful. 👍🏼

I completely agree 👍
Most labels should be delegated to the app and not be "computed" by UI Kit, so that we get good a11y labels and allow i18n customisation. We will review this behaviour.

Once https://github.com/lumada-design/hv-uikit-react/pull/3561 is merged, I'll close this then

**Comment by MEsteves22** [comment]
Hello 👋

Thank you for opening this issue! The problem with the dropdown menu was fixed on version [5.24.0](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-core%405.24.0).
